src/runtime/components/form/AntCheckboxWidget/AntCheckbox.vue

```diff
@@ -73,19 +74,28 @@ const inputClasses = computed(() => {
 
 const valueClass = computed(() => {
   const classes = {
-    'text-sm': true,
     'cursor-not-allowed opacity-50': props.disabled,
+		'text-sm': props.size === AntCheckboxSize.md,
+		'text-xs': props.size === AntCheckboxSize.sm
   };
 
   return classes;
 });
 
+const fieldSize = computed(() => {
+	if(props.size === AntCheckboxSize.md) {
```

Comment:
Einrückung prüfen + Semicolons fehlen bei den beiden Returns und in Zeile 91 Fehlt auch eins :)